cas/service.c

```diff
@@ -1326,6 +1373,35 @@ static int cas_fom_tick(struct m0_fom *fom0)
 			m0_ctg_op_init(&fom->cf_ctg_op, fom0,
 				       cas_op(fom0)->cg_flags);
 		}
+
+		/* If cas request has a txr payload, this a dtm0 operation.
+		 * We need to calculate credits for creating a dtm0 log record.
+		 *
+		 */
+		if (!m0_dtm0_tx_desc_is_none(&cas_op(fom0)->cg_txd)) {
+			struct m0_be_tx_credit dtm0logrec_cred;
```

Comment:
Is this needed to be initialized to ZERO?

Reply:
Sure.